code/modules/projectiles/ammo_types/heavy_ammo.dm

```diff
@@ -34,11 +34,11 @@
 /datum/ammo/bullet/minigun/ltaap
 	name = "chaingun bullet"
 	damage = 30
-	penetration = 15
-	sundering = 0
+	penetration = 35
+	sundering = 1
 	ammo_behavior_flags = AMMO_BALLISTIC|AMMO_IFF
-	damage_falloff = 2
-	accuracy = 80
+	damage_falloff = 1
+	accuracy = 90
```

Comment:
this is **plus** 90 accuracy, so it already has like a 180% chance to hit things, base.

Reply:
well thats dumb, bringing, I'll just not touch it then